presentation/src/main/java/com/globant/equattrocchio/cleanarchitecture/mvp/presenter/ImagesPresenter.java

```diff
@@ -23,53 +22,42 @@ public ImagesPresenter(ImagesView view, GetLatestImagesUseCase getLatestImagesUs
         this.getLatestImagesUseCase = getLatestImagesUseCase;
     }
 
-    public void onCountButtonPressed() {
-
-        view.showText(new String(""));//todo: aca va el string que me devuelva el execute del usecase
-
-
+    public void showText(String text) {
+        view.showText(text);
     }
 
     private void onCallServiceButtonPressed() {
 
-        getLatestImagesUseCase.execute(new DisposableObserver<Boolean>() {
+        getLatestImagesUseCase.execute(new DisposableObserver<String>() {
             @Override
-            public void onNext(@NonNull Boolean aBoolean) {
-                loadFromPreferences();
+            public void onNext(@NonNull String s) {
+                showText(s);
             }
 
             @Override
             public void onError(@NonNull Throwable e) {
-               view.showError();
+                view.showError();
             }
 
             @Override
             public void onComplete() {
                 new ImagesServicesImpl().getLatestImages(null);
             }
-        },null);
-
+        }, null);
 
 
         //todo acá tengo que llamar a la domain layer para que llame a la data layer y haga el llamdo al servicio
```

Comment:
Always use capitals for `TODO`. Also, this `TODO` will not be implemented here, so please remove the note.